<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>431</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="8" column="1">
     <widget class="QPushButton" name="new_btn2">
      <property name="text">
       <string>선택</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="new_btn3">
      <property name="text">
       <string>저장</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="new_btn1">
      <property name="text">
       <string>불러오기</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbl1_9">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>엑셀 파일 불러오기</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="new_lineEdit1"/>
    </item>
    <item row="11" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="new_startbtn">
        <property name="text">
         <string>실행</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="new_closebtn">
        <property name="text">
         <string>취소</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="new_lineEdit2"/>
    </item>
    <item row="8" column="0">
     <widget class="QComboBox" name="new_com2">
      <item>
       <property name="text">
        <string>환자정보</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>진단</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>수술</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>혈액검사</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>간호기록</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>조직검사</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>마취기록</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>미생물</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>영상검사</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>수술기록지</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>내시경검사</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>항암제</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>외래초진</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ASA</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lbl1_8">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DB 이름 설정하기</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="new_lbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="new_com1">
      <item>
       <property name="text">
        <string>sheet1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sheet2</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>새로운 DB 업로드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homebtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
